File: DbLib/organizer.cs
namespace DbLib
{
    using System;
    using System.Collections.Generic;
    using System.ComponentModel.DataAnnotations;
    using System.ComponentModel.DataAnnotations.Schema;
    using System.Data.Entity.Spatial;

    [Table("educationPlanner.organizers")]
    public partial class Organizer
    {
        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Usage", "CA2214:DoNotCallOverridableMethodsInConstructors")]
        public Organizer()
        {
            schoolings = new HashSet<Schooling>();
        }

        [Key]
        public int organizer_id { get; set; }

        [Required]
        [StringLength(255)]
        public string name { get; set; }

        [Required]
        [StringLength(255)]
        public string contact_person { get; set; }

        [Required]
        [StringLength(255)]
        public string email { get; set; }

        [Required]
        [StringLength(255)]
        public string website { get; set; }

        [Required]
        [StringLength(30)]
        public string phone { get; set; }

        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Usage", "CA2227:CollectionPropertiesShouldBeReadOnly")]
        public virtual ICollection<Schooling> schoolings { get; set; }
    }
}

File: DA/ClientApp/src/app/backend-detail/Organizer.ts
export class Organizer {
  id: Number;
  name: string;
  contactPerson: string;
  email: string;
  phone: string;
  website: string;
  constructor(id: Number,
    name: string,
    contactPerson: string,
    email: string,
    phone: string,
    website: string, ) {
    this.id = id;
    this.name = name;
    this.contactPerson = contactPerson;
    this.email = email;
    this.phone = phone,
      this.website = website;

  }
}
